kentikapi/internal/api_payloads/custom_application.go

```diff
@@ -41,9 +42,9 @@ type CustomApplicationPayload struct {
 	ASN         *string `json:"asn,omitempty"`
 
 	// following fields can appear in request: none, response: get/post/put
-	ID          *models.ID `json:"id,omitempty" response:"get,post,put"`
-	CompanyID   *models.ID `json:"company_id,string,omitempty" response:"get,post,put"`
-	UserID      *models.ID `json:"user_id,string,omitempty"`           // user_id happens to be returned as null
+	ID          *int       `json:"id,omitempty" response:"get,post,put"`
+	CompanyID   *models.ID `json:"company_id,omitempty" response:"get,post,put"`
```

Comment:
type should be: *string, here and everywhere else - payloads fields don't depend on our library public interface but rather on Kentik API server interface